building/build-acis/dnsmasq/dnsmasq.conf

```diff
@@ -0,0 +1 @@
+user=root
```

Comment:
Why do we need dnsmasq to run as root?

Reply:
It's inside a container. Running as root is a default.